proof-libs/fstar/core/Core.Hint.fsti

```diff
@@ -1,3 +1,4 @@
 module Core.Hint
 
-val black_box: #a:Type0 -> x:a -> y:a{y == x}
\ No newline at end of file
+val black_box: #a:Type0 -> x:a -> y:a{y == x}
+val must_use: #t:Type0 -> x:t -> y:t{y == x}
```

Comment:
Can you transform those two into lets? Those are really just the identity function, so let's make them normalizable.
e.g. `let must_use #t (x: t) = x` and same for `black_box`